### PR TITLE
Only throttle non-GET requests

### PR DESCRIPTION
**Pull request by meichstedt**

### CODE REVIEW

**Comment by meichstedt** [comment]
LGTM

File: src/main/scala/mesosphere/marathon/api/LimitConcurrentRequestsFilter.scala [hunk 1]
```diff
@@ -15,8 +15,22 @@ class LimitConcurrentRequestsFilter(concurrentOption: Option[Int]) extends Filte
   val filterFunction = concurrentOption.map(_ => withSemaphore _).getOrElse(pass _)
 
   def withSemaphore(request: ServletRequest, response: ServletResponse, chain: FilterChain): Unit = {
+    request match {
+      case r: HttpServletRequest if r.getMethod != "GET" =>
```
Comment (aquamatthias) [inline src/main/scala/mesosphere/marathon/api/LimitConcurrentRequestsFilter.scala]: Can you extend the test class, so we make sure GET requests are not blocked.

Comment (aquamatthias) [inline src/main/scala/mesosphere/marathon/api/LimitConcurrentRequestsFilter.scala]: I added a test case.

**Comment by aquamatthias** [comment]
LGTM

**Pull request merged**